object/types.proto

```diff
@@ -174,13 +178,17 @@ message Header {
     Header parent_header = 4 [json_name = "parentHeader"];
 
     // List of identifiers of the objects generated by splitting current one.
+    // Emtpy children list in a link object means that children's IDs
```

Comment:
here `link object` should refer to `SplitInfo.link` cuz that is where this entity is defined. This part is learned much earlier

Reply:
clarify your thoughts. `children` field only relates the Link object. a reader should have understood it somehow before. what do you mean by "should refer to SplitInfo.link"?

Reply:
> children field only relates the Link object.

i dont see this in `children` field comment. In current description this field has no connection with linking objects

> a reader should have understood it somehow before

and if he dont? Usually i see ref that allows me to become prepared

> what do you mean by "should refer to SplitInfo.link"

any ref that will allow reader to know _why_ this field is actually needed. Ref to `SplitInfo.link` will make it clear that the field is present in and only in linking objects

Reply:
> i dont see this in children field comment

it always was that way and the comment was not clear always

>> a reader should have understood it somehow before
>
> and if he dont?

then it is sad. like was before this PR

>  Ref to SplitInfo.link

what is ref to smth? like in go pkg? with `[]`? i did not find any examples: 
```
▶ grep -r -E "// .+\[" .
./acl/types.proto:// like [JWT](https://jwt.io), it has a limited lifetime and scope, hence can be
./netmap/types.proto:  //   [UN/LOCODE](https://www.unece.org/cefact/codesfortrade/codes_index.html)
./netmap/types.proto:  //   [ISO 3166-1_alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
./netmap/types.proto:  //   [ISO-3166](https://www.iso.org/obp/ui/#search). Calculated automatically
./netmap/types.proto:  //   Presented in [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2)
./netmap/types.proto:  //   [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2). Calculated
./netmap/types.proto:  //   Node's continent name according to the [Seven-Continent model]
./accounting/types.proto:// Please see [The General Decimal Arithmetic
./refs/types.proto:// String presentation is a [Base58
./status/types.proto:    //  - [**0**] Magic number of the served NeoFS network (big-endian 64-bit
./status/types.proto:    // - [**0**] Human-readable description (UTF-8 encoded string).
./reputation/service.proto:        // floating point number in [0;1] range.
./reputation/types.proto:    // Trust level in [0:1] range
```

Reply:
> it always was that way and the comment was not clear always

that's why i suggested clarifications - to make it clearer

> what is ref to smth?

reference

Reply:
> reference

i meant, how do you see it? how should it look like?

Reply:
plain text

btw
```suggestion
    // Empty children list in a link object means that children's IDs
```

Reply:
fixed typo

> plain text

well, i tried. i dont know what a ref is in our API. we haven't done it yet